closing scrum -> demo day (#23)
greenelab/onboarding: onboarding.md

**Scrum:** Our team's scrum process involves three components:

    1. An opening scrum meeting at 9:45 AM Monday morning where individuals will lay out their goals for the week, in the form of post-it notes, on the whiteboard in the Greene Lab.

>     2. A demo day meeting at 1:30 PM Friday afternoon where team members show off an accomplishment from the week in 3 minutes or fewer. This could be a new figure, section of a paper, or some code that they are particularly happy with.

Author: ```suggestion
    2. A demo day meeting at 1:30 PM Friday afternoon where team members show off an accomplishment from the week in 3 minutes or fewer. This could be a new figure, section of a paper, some code that they are particularly happy with, or something that we learned from a paper, poster, or research presentation.
```

Author: Thanks @ajlee21!

    3. A daily virtual scrum update.

> We use [a Google Slides deck](https://docs.google.com/presentation/d/1JqlxrxBJUYoA3kxaVnjbAjbL1pOU1rM5N4tYaQ5uAb8/edit?usp=sharing) to share figures or paper sections for the demo day meeting - if you are not already listed, feel free to add a slide with your name. Alternatively, screen sharing is possible for code demos or interactive weekly accomplishments. 

Reviewer: ```suggestion
We use Google Slides to share figures or paper sections for the demo day meeting; the link is pinned in the `#lab-meeting` channel on Slack and is not intended to be shared outside the lab. If you are not already listed in the slides, feel free to add a slide with your name. Alternatively, screen sharing is possible for code demos or interactive weekly accomplishments. 

```
After discussing in Slack, this seemed to be the consensus. Hopefully, this note will help new lab members find the link without needing a current member to remember to send it, and the slides will remain internal to the lab.

**Scrum:** Our team's scrum process involves three components:

    1. An opening scrum meeting at 9:45 AM Monday morning where individuals will lay out their goals for the week, in the form of post-it notes, on the whiteboard in the Greene Lab.

>     2. A demo day meeting at 1:30 PM Friday afternoon where team members show off an accomplishment from the week in 3 minutes or fewer. This could be a new figure, section of a paper, or some code that they are particularly happy with.

Reviewer 2: Do we also want to include something we've learned (i.e. similar to Halie's demo, or I'm thinking if we went to a conference and found some poster/talk interesting)?

>     2. A demo day meeting at 1:30 PM Friday afternoon where team members show off an accomplishment from the week in 3 minutes or fewer. This could be a new figure, section of a paper, or some code that they are particularly happy with.
>     3. A daily virtual scrum update.
> 
> The daily virtual scrum update should include an update to [scrum repository](https://github.com/greenelab/scrum).

Reviewer: Bumping this PR since @axiomcura was asking about the demo day slides - it would be good to have a link in the onboarding doc for new lab members in the future.

```suggestion
    2. A demo day meeting at 1:30 PM Friday afternoon where team members show off an accomplishment from the week in 3 minutes or fewer. This could be a new figure, section of a paper, or some code that they are particularly happy with.
    3. A daily virtual scrum update.

We use [a Google Slides deck](https://docs.google.com/presentation/d/1JqlxrxBJUYoA3kxaVnjbAjbL1pOU1rM5N4tYaQ5uAb8/edit?usp=sharing) to share figures or paper sections for the demo day meeting - if you are not already listed, feel free to add a slide with your name. Alternatively, screen sharing is possible for code demos or interactive weekly accomplishments. 

The daily virtual scrum update should include an update to [scrum repository](https://github.com/greenelab/scrum).

```
